tools/cldr/cldr-to-icu/pom.xml

```diff
@@ -24,7 +24,7 @@
             for the icu4j.version tag to use. In general we should just use the latest
             SNAPSHOT for the ICU version that we want, so this should only need updating
             when the ICU version changes e.g. from 74.0.1, to 74.1, then to 75.0.1 -->
-        <icu4j.version>78.1</icu4j.version>
+        <icu4j.version>78.2</icu4j.version>
         <cldr-code.version>48.0</cldr-code.version>
```

Comment:
@mihnita `cldr-code.version` should be "48.1-SNAPSHOT", that is what it was in my original PR, but I guess that got changed when the pom conflict was resolved. Maybe I can fix that....(but not until tomorrow)

Reply:
Thanks!